### PR TITLE
chore(design-system): browser matrix for Cypress

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**

Cypress executes tests using Electron, that's sad since it's designed to be run through real web browsers!

**What is the chosen solution to this problem?**

Run E2E tests on Chrome and Firefox in parallel 💯 

**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: .github/workflows/design-system-component-testing.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ on:
 jobs:
   cypress-run:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```
Comment (frassinier) [inline .github/workflows/design-system-component-testing.yml:13]: If tests fail on Chrome, tests on Firefox must continue, and the opposite is truthy too.

**Review by lmaillet — APPROVED** [review]

File: packages/design-system/README.md [hunk 2]
```diff
@@ -60,7 +60,8 @@ Limit changes to styled-components scope.
 
 ### End-to-End tests
 
-Prior to end-to-end tests if you want to test interactions in real browsers.
+Simple renders will be covered by Chromatic.
+Prior to end-to-end tests, using Cypress, if you have to test interactions in real browsers.
```
Comment (Benrajalu) [inline packages/design-system/README.md:64]:
```suggestion
Prior to end-to-end tests, using Cypress, if you have to test interactions in real browsers.
```
This doesn't make grammatical sense.

Comment (jsomsanith-tlnd) [reply packages/design-system/README.md:64]: +1, don't understand what you want to say here.
The rest LGTM

Comment (jsomsanith-tlnd) [reply packages/design-system/README.md:64]: +1, don't understand what you want to say here.
The rest LGTM

**Review by jsomsanith-tlnd — APPROVED** [review]

**Review by jmfrancois — COMMENTED** [review]
Did you test it ? I would be interested to see the upload result :)
https://github.com/Talend/ui/actions/runs/1786406539

**Review by Benrajalu — APPROVED** [review]

**Comment by frassinier** [comment]
@jmfrancois yes, the path was wrong, I've fixed it this morning 😇 
https://github.com/Talend/ui/actions/runs/1788441373

**Pull request merged**